### PR TITLE
enable JSON posts

### PR DESCRIPTION
**Pull request by Yall1963**
Hello Esri,

I've encoutered problems with your resource-proxy when trying to handle post requests containing an 'application/json' body. The sympotom was that these requests were treated as get requests and the JSON data was lost.
So I've forcked your project to add support for this scenario.

BTW: This is my very first Pull request ever. So please tell me if I'm doing anything wrong ;-)

Regards,
Yall1963

### CODE REVIEW

File: PHP/proxy.php [hunk 1]
```diff
@@ -5,6 +5,8 @@
  *
  * Version 1.1.0
  * See https://github.com/Esri/resource-proxy for more information.
+ * 
+ * forked by Yall1963 to support POST requests including a JSON body
```
Comment (jgravois) [inline PHP/proxy.php]: no need for this if we plan on merging your changes back into our own project

**Comment by bsvensson** [comment]
Thanks @Yall1963 for helping making the resource-proxy better.

Here's a few questions:
* Could you talk more about the problem and how we can reproduce it?  Both so we can test your fix and also see if the DotNet and JSP proxies have the same problem.
* The composer.json addition doesn't seem related to fixing the json issues. Could you remove that from the pull request?  Or make a separate pull request if you think that is something of value to other people using the PHP proxy.

**Comment by jgravois** [comment]
@bsvensson the content of the composer.json commit seems to already have been reverted.  i can help @Yall1963 w/ squashing/rebasing **after** all other concerns have been attended to via additional commits in this PR.

**Comment by phpmaps** [comment]
Thanks for this.  Would you include a GET / POST request showing the problem or sample JS / HTML file showing the problem?



Sent via the Samsung GALAXY S® 5, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by bsvensson** [comment]
Following up on @jgravois help offer from November:
> i can help @Yall1963 w/ squashing/rebasing after all other concerns have been attended to via additional commits in this PR.

@Yall1963 - any progress on this?

If not, @jgravois do you what the essence of this solution was? If not, then I'd suggest we close this pull request.

**Comment by jgravois** [comment]
i have no clue how @Yall1963 solved the problem and agree that if we don't hear from him soon, closing this PR is the best course of action.

that said, if he was seeing behavior in which POST requests were incorrectly passed along as GETs (stripping out `application/json` content in the request body) we should probably consider creating a new issue and attempting to reproduce the error.

**Comment by dasa** [comment]
There's two commits made to proxy.php at: https://github.com/Yall1963/resource-proxy/commits/master/PHP/proxy.php

**Comment by bsvensson** [comment]
Closing this old PR.

**Pull request closed without merging**